R/splatPop-simulate.R

```diff
@@ -145,7 +145,7 @@ splatPopSimulateMeans <- function(vcf=mock_vcf(),
     checkmate::assertClass(params, "splatPopParams")
     set.seed(getParam(params, "seed"))
     
-    nGroups <- getParam(params, "nGroups")
+    nGroups <- length(getParam(params, "group.prob"))
```

Comment:
Why did `nGroups` not work? They should be the same thing... Just wondering if there is a bigger issue here we might need to look at.

Reply:
Groups wasn't working for me because I hadn't included all of your checks to setMethod("setParam"), so updating group.prob wasn't automatically changing the nGroup. I have fixed that now and reverted this code back to calling nGroups for simplicity.